src/data/agile-quiz.ts

```diff
@@ -175,6 +175,17 @@ const agileQuiz = [
       "Planning poker is an estimation and planning technique in the Agile development model. It is used to estimate the development effort required for a user story or a feature.",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/",
   },
+  {
+    Question:
+      "In which environment does Agile works great?",
+    Answer: "It works great in dynamic environments where there is a potential for changing or evolving requirements.",
```

Comment:
It is best to keep answers short
```suggestion
    Answer: "It works great in dynamic environments",
```